new-runtimepoc/Pester.Runtime.psm1

```diff
@@ -1057,9 +1058,12 @@ function Run-Test {
     [CmdletBinding()]
     param (
         [Parameter(Mandatory = $true)]
-        [PSTypeName("DiscoveredBlock")][PSObject[]] $Block,
```

Comment:
Rest of the changes in this file are just unnecessary formatting changes, please revert :)

Reply:
Done